### PR TITLE
Check whether variant is empty when uploading files

### PR DESCRIPTION
**Pull request by Olgabrezel**

### CODE REVIEW

File: Werewolf for Telegram/Werewolf Control/Helpers/LanguageHelper.cs [hunk 1]
```diff
@@ -182,6 +182,10 @@ internal static void UploadFile(string fileid, long id, string newFileCorrectNam
                 newFileErrors.Add(new LanguageError(newFile.FileName, "*Language Node*",
                     $"ERROR: The following file partially matches the same language node. Please check the file name, and the language name, base and variant. Aborting.\n\n*{error.FileName}.xml*\n_Name:_{error.Name}\n_Base:_{error.Base}\n_Variant:_{error.Variant}", ErrorLevel.Error));
             }
+            
+            //check whether variant is empty
+            if (newFile.Variant == "" || newFile.Variant == " ") newFileErrors.Add(new LanguageError(newFile.FileName, "*Language Node*",
```
Comment (renyhp) [inline Werewolf for Telegram/Werewolf Control/Helpers/LanguageHelper.cs]: Would be better to use `string.IsNullOrWhiteSpace(newFile.Variant)`

**Comment by Olgabrezel** [comment]
Lel, I never heared of that function before :D

File: Werewolf for Telegram/Werewolf Control/Helpers/LanguageHelper.cs (hunk 1 above)
Comment (Olgabrezel) [inline Werewolf for Telegram/Werewolf Control/Helpers/LanguageHelper.cs]: I didn't know this function existed yet :D

**Comment by renyhp** [comment]
eh sorry, this was to be done differently... some similar check already existed, and that one was to fix. See b029796. Closing this PR

**Pull request closed without merging**